### ISSUE TITLE
Please update comment

### ISSUE THREAD

**Issue by sahasomnath** [rssiweb/attendance#18]
![image](https://user-images.githubusercontent.com/26809222/40899023-39a5027c-67e2-11e8-8050-96488cc20e6b.png)
![image](https://user-images.githubusercontent.com/26809222/40899029-52fdaefe-67e2-11e8-9c0f-a13caed3373d.png)

**Comment by zkhan93** [comment]
admin can add/update marks from admin section, also he can update comment and marks range, please look at that feature.

**Comment by sahasomnath** [comment]
hey Zee, there is no issue from editing perspective. its data issue. Comments are not as per the expectation.

**Comment by zkhan93** [comment]
Yes, it is a data issue. And the Admin is in-charge of this data not the developer

**Comment by zkhan93** [comment]
the comments are populated only once, next time if you update the marks comments have to be updated manually, if you want the comments to be replaced by the default comments according to the new marks do let me know.

**Comment by sahasomnath** [comment]
Zee listen me please. Data issue is a different thing. What I'm saying, under admin section data is correct and it's my responsibility but  is it coming to my action item if that data is not showing under the exam screen. Also, yes you are correct. The comment section is editable but it should always display the comments as per the current value under marks. Not based on very first marks.
